README.md

```diff
@@ -34,8 +48,7 @@ _Requires node version >= v11.15.0_
 9. Set `NEXT_PUBLIC_MIXPANEL_DEV` to the valid Mixpanel dev environment API key
 10. Set `NEXT_PUBLIC_MIXPANEL_PROD` to the valid Mixpanel prod environment API key
 11. Set `GOERLI_FORK_API_KEY` for the API key
-12. Set `ETHERSCAN_KEY` for ethers provider to use
```

Comment:
hey @ultraschuppi , i would leave the ETHERSCAN_KEY on the .env variables config section too

Reply:
good point, shall i just revert the last commit?

Reply:
Yes, if you could that'd be great. Then we can merge.

Reply:
done